src/duration.rs

```diff
@@ -742,38 +742,76 @@ impl Duration {
 // region: trait impls
 /// The format returned by this implementation is not stable and must not be relied upon.
 ///
+/// By default this produces an exact, verbose printout of the duration.
+/// For a concise, rounded printout instead, you can use the `.N` format specifier:
+///
+/// ```
+/// use time::Duration;
```

Comment:
This should be hidden (by starting the line with `#`). This is because the import is implied — it's being shown in the `Duration` docs. The following blank line should either be removed or hidden as well.